relay-event-normalization/src/normalize/mod.rs

```diff
@@ -716,79 +680,6 @@ mod tests {
         assert_eq!(dynamic_config.max_custom_measurements().unwrap(), 3);
     }
 
-    #[test]
-    fn test_handles_type_in_value() {
```

Comment:
These tests have been moved to `event.rs`.

This one in particular, `test_handles_type_in_value`, was testing two things, so I split it into `test_handle_types_in_spaced_exception_values` and `test_handle_types_in_non_spaced_excepton_values`.